README.md

```diff
@@ -42,13 +44,11 @@ The documentation is generated using MkDocs. For full documentation visit [mkdoc
         index.md  # The documentation homepage.
         ...       # Other markdown pages, images and other files.
 
-#### Commands
+In order to build `OpenSquirrel` documentation, run the following command in the root directory of `OpenSquirrel`:
 
-* `mkdocs new [dir-name]` - Create a new project.
-* `mkdocs serve` - Start the live-reloading docs server.
-* `mkdocs build` - Build the documentation site.
-* `mkdocs -h` - Print help message and exit.
-* `mkdocs gh-deploy` - Rebuild and deploy documentation to GitHub Pages.
+```shell
+mkdocs build
+```
```

Comment:
This is information for the developers of OpenSquirrel, they will need to know how to use MkDocs, any idea if there is another place we should keep this information? Makes sense not to have it in the README.md.

Reply:
Thanks for the comment, Chris.

I have a few personal `cheat sheet` guides that would inded better be in a share point, accessible to other engineers. For example, for `git`, for `docker`, for `OpenQL`/`libqasm`/`QX-simulator` builds, and so on. Typical operations you do once in a blue moon and you don't want to memorize. We could well have an `MkDocs` cheat sheet as well. What do you think?

Now, if for OpenSquirrel doc managing, you think that the developer would need to execute a few commands, I would, instead of just writing down a short manual, write a list of steps. E.g., for building OpenSquirrel docs, run `mkdocs new...`, then `mkdocs serve`, etc. I just assumed that an OpenSquirrel developer would only need to do the `build` step.

Reply:
My point is:
- We could put it here, but I wouldn't just write a short MkDocs reference. I would instead write a set of steps that the OpenSquirrel developer may need to use, as I pointed out in my comment above.
- If that section was too large, we could link from here to a `docs/README.md`, where we could be more verbose because we would only talk about MkDocs there.